dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -322,37 +328,102 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         co_await winrt::resume_background();
 
         std::wstring appId{ RetrieveNotificationAppId() };
+
         THROW_IF_FAILED(ToastNotifications_RemoveAllToastsForApp(appId.c_str()));
     }
 
+#if 1 //  Option 1
     winrt::Windows::Foundation::IAsyncOperation<winrt::Windows::Foundation::Collections::IVector<winrt::Microsoft::Windows::AppNotifications::AppNotification>> AppNotificationManager::GetAllAsync()
     {
-        co_await winrt::resume_background();
+        HRESULT telemetryResult{ S_OK };
```

Comment:
Just call it hr.

winrt::hresult hr{S_OK};

Reply:
done